test/e2e_env/kubernetes/meshhttproute/test.go

```diff
@@ -18,6 +18,29 @@ func Test() {
 	meshName := "meshhttproute"
 	namespace := "meshhttproute"
 
+	kubeMeshServiceYAML := fmt.Sprintf(`
+apiVersion: kuma.io/v1alpha1
```

Comment:
Does it support MeshGateway in top level? If so, let's add a test to cover it.

Reply:
this does not cover `MeshGateway` in top level targetRef, as we don't build resource rules for MeshGateway. I will create separate issue to cover this